### DISCUSSION TITLE
a way to add a location like models in yaml but for workflows?

### DISCUSSION THREAD

**Discussion by tampadesignr** [comfyanonymous/ComfyUI#5793] [Q&A] [answered]
a way to add a location like models in yaml but for workflows?
or a simple place to add a bit of code

i like to test nodes on a separate portable install and would like to point each install to the same workflow folder
thank you

**Comment by ltdrdata** [comment]
You can use  --user-directory command line option.

**Reply by tampadesignr** [reply]
> You can use --user-directory command line option.

so  i do it like this? thanks btw

--output-workflows   G:\myworksfolder\workflows

**Reply by ltdrdata** [reply, marked as answer]
No, such an option does not exist.  

The `--user-directory` option is used to configure the location for saving and reading user data somewhere other than the default `ComfyUI/users`. Workflow files are also stored within this directory.

**Reply by tampadesignr** [reply]
> No, such an option does not exist.
> 
> The `--user-directory` option is used to configure the location for saving and reading user data somewhere other than the default `ComfyUI/users`. Workflow files are also stored within this directory.

 --user-directory G:\ComfyUI\ComfyUI\user
so its this then?
i am a old school artist just want to create... coding is required..uughhhh
--- tried and worked thank you
